### PR TITLE
Plugin endpoint

### PR DESCRIPTION
**Pull request by PierreLeGuen**
* Added endpoints
* Added doc

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -148,7 +148,26 @@ client.website.send_message_in_conversation(
...
+  * **List All Active Subscriptions**: `client.list_all_active_subscriptions()`
+  * **List Subscriptions For Website**: `client.list_subscriptions_website(website_id)`
+  * **Get Subscription Details**: `client.get_subscription_details(website_id, plugin_id)`
+  * **Subscribe Website To Plugin**: `client.subscribe_plugin_to_website(website_id)`
+  * **Unsubscribe Plugin From Website**: `client.unsubscribe_plugin_from_website(website_id, plugin_id)`
+  * **Get Subscription Settings**: `client.get_subscription_settings(website_id, plugin_id)`
+  * **Save Subscription Settings**: `client.save_subscription_settings(website_id, plugin_id, settings)`
+  * **Update Subscription Settings**: `client.update_subscription_settings(website_id, plugin_id, settings)`
+  * **Forward Plugin Payload To Channel**: `client.forward_plugin_payload_to_channel(website_id, plugin_id, data)`
+  * **Dispatch Plugin EventPOST**: `client.dispatch_plugin_event(website_id, plugin_id, data)`
```
Comment (eliottvincent) [inline README.md:169]: Typo, remove `POST`

File: README.md [hunk 2]
```diff
@@ -148,7 +148,26 @@ client.website.send_message_in_conversation(
...
   * **Count Visitors**: `client.website.count_visitors(website_id)`
   * **List Visitors**: `client.website.list_visitors(website_id, page_number)`
-
+  
+### Plugin
+
+* **Plugin Connect**
+  * **Resolves the current plugin account information**: `client.get_connect_account()`
+  * **Checks whether the connected plugin session is valid**: `client.check_connect_session_validity()`
+  * **Lists all websites linked to connected plugin**: `client.list_all_connect_websites(page_number, filter_configured)`
```
Comment (eliottvincent) [inline README.md:157]: Rename descriptions as per https://github.com/crisp-im/node-crisp-api/commit/ae9bfcffd3cbfe2e61aae564844a1c40ab7d1316

**Review by eliottvincent — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (PierreLeGuen) [reply README.md]: Addressed comment in https://github.com/crisp-im/python-crisp-api/pull/10/commits/a66079e15946702b57945822036487dbd918524d.

File: README.md (hunk 2 above)
Comment (PierreLeGuen) [reply README.md]: Addressed comment in https://github.com/crisp-im/python-crisp-api/pull/10/commits/a66079e15946702b57945822036487dbd918524d.

**Pull request merged**